docs/get-involved/run-node/paratime-node.mdx

```diff
@@ -112,6 +112,40 @@ committee on Mainnet, your entity needs to:
 - Have a validator in the **validator set**.
 - Have at least **5,000,000.00 ROSE staked/delegated** to it.
 
+</TabItem>
+<TabItem value="Sapphire ParaTime">
```

Comment:
Let's change the order of tabs so it's consistent with https://docs.oasis.io/dapp/:
1. Sapphire
2. Emerald
3. Cipher